### DISCUSSION TITLE
[1 febbraio 2022 - A] Esercizio 5

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#209] [Esami]
**Esercizio 5 (2 punti)**

Si disegni il diagramma temporale dei segnali **y0, y1 e z** del seguente circuito a partire dallo stato **y0=1** e **y1=1** quando riceve in ingresso la sequenza di bit $01001$.

![image](https://user-images.githubusercontent.com/69389461/211830307-8b2061aa-2054-48fb-acd0-8d4b305190c4.png)

**Comment by Elia-Belli** [comment]
$y_0' = x \oplus y_1$
$y_1' = y_0 + \bar x$
$z = y_1 \bar x$
| | | | | | ||
|---|---|---|---|---|---|---|
| $x$ | $X$ | $0$ | $1$ | $0$ | $0$ | $1$ |
| $y1$ | $1$ | $1$ | $1$ | $1$ | $1$ | $1$|
| $y0$ | $1$| $1$ | $0$ | $1$ | $1$ | $0$|
| $z$ | $X$ | $1$ | $0$ | $1$ | $1$ | $0$ |

**Comment by fra01asr** [comment]
Ciao scusa,
 y1 non è Y0 + !x?

**Comment by fraxb04** [comment]
@Elia-Belli 
allego la forma d'onda:
![image](https://user-images.githubusercontent.com/122573871/216782442-b92ddf37-34c5-474c-90a0-e13f028587b9.png)
